Makefile

```diff
@@ -107,10 +107,10 @@ generate-deepcopy: ## Runs controller-gen to generate deepcopy files
 .PHONY: generate-manifests
 generate-manifests: ## Generate manifests e.g. CRD, RBAC etc.
 	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go \
-		paths=./pkg/... \
+		paths=./api/... \
```

Comment:
shouldn't this also include the controllers directory as well?

Reply:
Added!